research/sentiment_analysis/sentiment_main.py

```diff
@@ -1,160 +1,79 @@
-"""The main module for sentiment analysis.
-
-The model makes use of concatenation of two CNN layers
-with different kernel sizes.
-See `sentiment_model.py` for more details about the models.
-"""
```

Comment:
Why remove the module docstring?

Reply:
I am adding doc string back to the main module also adding to other modules.